utils/format.go

```diff
@@ -508,19 +508,28 @@ func FormatHash(hash []byte, trunc_opt ...bool) template.HTML {
 	return template.HTML(fmt.Sprintf("<span class=\"text-monospace\">%#x</span>", hash))
 }
 
+func formatWithdrawalHash(hash []byte) template.HTML {
+	colorClass := "text-warning"
```

Comment:
It would require a
```
colorClass := ""
if A {
    colorClass = "X"
} else {
    colorClass = "Y"
}
```

If you prefer that, I am fine with changing it.

Reply:
`var colorClass string` would be better to read